templates/databricks-multi-workspace.template.yaml

```diff
@@ -204,30 +207,32 @@ Parameters:
     MaxLength: '63'
     Type: String
     ConstraintDescription: The Quick Start bucket name can include numbers, lowercase letters, uppercase letters, and hyphens (-). It cannot start or end with a hyphen (-).
-  KeyArn:
-    Description: "(Optional) AWS KMS key ARN to encrypt and decrypt the workspace notebooks in the control plane. Set only if using the feature customer-managed key for notebooks. See https://docs.databricks.com/security/keys/customer-managed-keys-notebook-aws.html."
-    Default: ''
-    Type: String
-  KeyAlias:
-    Description: "(Optional) AWS KMS key alias. Keep blank if you didn't specify a key ARN."
-    Default: ''
-    Type: String 
-  KeyRegion:
-    Description: "(Optional) AWS Region of your AWS KMS key. Keep blank if you didn't specify a key ARN."
-    Default: ''
-    Type: String
   VPCID:
-    Description: "(Optional) ID for your VPC in which to create the new workspace. Set only if using the customer-managed VPC feature. The format is vpc-xxxxxxxxxxxxxxxx. If unspecified, Databricks creates the new workspace in a new VPC that Databricks creates. See https://docs.databricks.com/administration-guide/cloud-configurations/aws/customer-managed-vpc.html."
+    Description: "ID for your VPC in which to create the new workspace. Set only if using the customer-managed VPC feature. Format is vpc-xxxxxxxxxxxxxxxx. See https://docs.databricks.com/administration-guide/cloud-configurations/aws/customer-managed-vpc.html. If unspecified, Databricks creates the new workspace in a new VPC that Databricks creates."
```

Comment:
Same comment as other template. Are these parameters no more optional?

Reply:
similar to all Optional parameters. These sections are already labelled as Optional